File: WebApi/WebApp.DataContext/WebApp/Policy.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WebApp.DataContext.WebApp
{
    public class Policy
    {
        public int PolicyId { get; set; }
        public string PolicyName { get; set; }
        public string PolicyDescription { get; set; }
        public int PolicyPeriod { get; set; }
        public long PolicyPrice { get; set; }
        public int RiskTypeId { get; set; }
        public int CoverageTypeId { get; set; }
        public System.DateTime PolicyStartDate { get; set; }

        public virtual CoverageType CoverageType { get; set; }
        public virtual RiskType RiskType { get; set; }
    }
}

File: FrontEnd/src/app/_models/Policy.ts
export class Policy {

    public constructor(init?: Partial<Policy>) {
        Object.assign(this, init);
    }

    PolicyId: number;
    PolicyName: string;
    PolicyDescription: string;
    PolicyPeriod: number;
    PolicyPrice: number;
    RiskTypeId: number;
    CoverageTypeId: number;
    PolicyStartDate: string;    
}